RELEASE_TEMPLATE.md

```diff
@@ -0,0 +1,253 @@
+## Subject Line
+[Blox] Deploy release - v$$blox_version$$
+
+## Activity Details
+This activity is to: Release Blox version v$$blox_version$$ by merging the dev branch into the master branch on GitHub, and pushing new Blox images up to Docker Hub.
+
+The purpose of this change is to: Release the latest features and bug fixes for Blox to allow consumers to start using the new functionality.
+
```

Comment:
I'd prefer to leave the specific changes in the CHANGELOG, and keep this a more generic template.